web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -173,6 +175,11 @@ public void init(FilterConfig config) {
 
             telemetryClient = new TelemetryClient(configuration);
             webModulesContainer = new WebModulesContainer(configuration);
+
+            if (StringUtils.isNotEmpty(config.getFilterName())) {
```

Comment:
Is this the fix that you are attempting to make for starter scenario when we inject filter?

Reply:
No, this is to fix the log messages.

Reply:
added comment

Reply:
I don't see it :)